docs/cases/api/cases-api/cases-api-update-comment.asciidoc

```diff
@@ -27,13 +27,16 @@ A JSON object with the updated comment:
 |`alertId` |String |The updated alert id. |Yes, only when type is `alert`.
 |`index` |String |The updated alert index. |Yes, only when type is `alert`.
 |`id` |String |The comment's ID. |Yes
+|`owner` |String |The application that owns the case. |Yes
+
+Must be a valid application owner that is registered within the Cases Role Based Access Control system (i.e. `securitySolution`). The value must also match the `owner` field of the case this comment will be attached to.
```

Comment:
Thanks for the feedback, RBAC isn't a term that Elastic coined https://en.wikipedia.org/wiki/Role-based_access_control would you prefer that it is lowercase?

Reply:
Yes, lowercase please. :) And I wanted to flag this just in case we needed to add a new acronym or term definition to the security docs. Recommend changing it to: case's role-based access control system.